modules/ROOT/pages/monitoring/concepts.adoc

```diff
@@ -0,0 +1,46 @@
+= Monitoring concepts
+:description: This section describes the concepts that are important to understand how Neo4j Ops Manager can be used for monitoring systems.
+
+This section describes the concepts that are important to understand how Neo4j Ops Manager can be used for monitoring systems in an optimal way.
+
+== Conditions
+
+A **Condition** specifies the constraints when the monitored entity reaches a certain state.
+Conditions are periodically evaluated, evaluation results are persisted.
+
+There are two **types** of Conditions:
+
+* **Event-based Conditions**: a Condition evaluates to **true** when a specific event was generated by the monitoring subsystem.
+Examples: "Unreachable/Offline agents detected", "Incompatible agent connected".
+* **Metric-based Conditions**: a Condition evaluates to **true** when the metric(s) assigned to it change in a way, so that its expression evaluates to **true**.
```

Comment:
```suggestion
* **Metric-based Conditions**: a Condition evaluates to **true** when the metric(s) assigned to it change in a way so that its expression evaluates to **true**.
```